scripts/gen_lazy_tensor.py

```diff
@@ -61,11 +61,113 @@ def node_base_ctor_call(self, schema: LazyIrSchema) -> str:
     scalar_hashes = ", ".join([f"{a.name}" for a in scalar_args])
 
     return f"""{self.node_base}(torch::lazy::OpKind({aten_symbol(schema)}),
-              {{{base_ctor_value_args}}}, std::move(shapes),
+              {{{base_ctor_value_args}}},
               [&]() {{ return {schema.node_name}OutputShape({shape_fn_inputs}); }},
               /* num_outputs */ {len(schema.returns)},
               torch::lazy::MHash({scalar_hashes}))"""
 
+  def gen(self, schema: LazyIrSchema) -> List[str]:
```

Comment:
Most of this function is same as upstream codegen. This is updated to remove `torch::lazy::Shape` related code.

Reply:
It looks like maybe we can make the parts we don't need as a method in the `GenLazyIR` class such that we can substitute them with things that output empty strings for instances.

Reply:
You did a good job on GenXlaLazyNativeFuncDefinition. We can repeat that.

Reply:
Thanks for the comment. It seems to be a bit harder to do make this overriding clean for the `GenLazyIR` class. The upstream `GenLazyNativeFuncDefinition` class has specific function named `shape_inference()` that we can override to remove the `torch::lazy::Shape` related logic. But for the upstream `GenLazyIR` class, the shape inference logic is part of a big `gen()` function so we are unable to override only the specific `torch:lazy:Shape` related codegen. I've added a comment to this function describing this, let me know if you think this is okay.

Reply:
Sorry, accidentally deleted one of your original comments on this thread while I was trying to delete one of my older replies lol. Just for future reference, Jiewen's original comment was:
```
It looks like maybe we can make the parts we don't need as a method in the GenLazyIR class such that we can substitute them with things that output empty strings for instances.
```